packages/pds/src/db/index.ts

```diff
@@ -127,9 +127,10 @@ export class Database {
   async close(): Promise<void> {
     this.channelClient?.removeAllListeners()
     this.channelClient?.release()
-    if (this.cfg.dialect === 'pg') {
-      await this.cfg.pool.end()
-    }
+    // @TODO investigate
```

Comment:
I snuck this in here, because it was causing tests to fail for some reason. Don't remember it causing issues on my branch 🤔 

Needs further investigation